src/main/java/com/xceptance/neodymium/module/statement/browser/BrowserStatement.java

```diff
@@ -285,6 +289,17 @@ private boolean canReuse(boolean preventReuse, WebDriverStateContainer webDriver
             browserAnnotations.addAll(classBrowserAnnotations);
         }
 
+        if (!classRandomBrowserAnnotation.isEmpty() && methodBrowserAnnotations.isEmpty())
```

Comment:
Replacement of those two statements with `if(){}else if(){}` structure and removal of one part of the first condition would change the expected behavior.

1. Removal of one part of the first condition:
Using the `@RandomBrowsers` annotation, I'd expect that it's possible to overwrite the `@RandomBrowsers` from the class level with simply `@Browser` annotation on the method level. This is why we need to check, whether the method has the `@Browser`. 

2. `else if` instead of two `if`s:
Analog to most of the other annotations in the Neodymium, the `@RandomBrowsers` annotation of the class level should be overwritable on the method level. Using `else if`  destroys this principle because if the annotation is present for class, the `else` wouldn't allow us to check if it should be overwritten on the method level.

Please let me know if the described behavior is not the desired one or if I got your comment wrong.